src/components/NotificationsTable/NotificationsTable.js

```diff
@@ -15,6 +15,7 @@ import {
 import './style.css'
 
 const NotificationsTable = ({ notifications, animateIncomplete }) => {
+    const done = notifications.some(n => n.completed)
```

Comment:
When the latest notification is marked as completed, earlier notifications do not have their `completed` field also set to true and so we must only render a busy/loading indicator if no notification is done.

Reply:
Is this being tested somehow?